plugin/src/main/java/io/jenkins/plugins/analysis/core/steps/PublishIssuesStep.java

```diff
@@ -884,20 +411,19 @@ protected ResultAction run() throws IOException, InterruptedException, IllegalSt
             }
             report.addAll(step.reports);
 
-            StageResultHandler statusHandler = new PipelineResultHandler(getRun(),
-                    getContext().get(FlowNode.class));
+            var deltaCalculator = DeltaCalculatorFactory
+                    .findDeltaCalculator(step.scm, getRun(), getWorkspace(), getTaskListener(), new FilteredLog());
```

Comment:
Requiring a workspace here seems to be an incompatible change: a previously working pipeline that collected a list from multiple parallel agents using `scanForIssues` and finally published them together using `publishIssues` is now broken. Can the workspace be made optional?

Reply:
It looks like the GitClient accepts `null` as workspace. So it should be possible, can you please file an issue in Jira?

Reply:
#1727

Reply:
@uhafner can you take a look please? This is a critical regression in the current plugin release.